src/crypto/index.js

```diff
@@ -468,7 +468,15 @@ Crypto.prototype.bootstrapSecretStorage = async function({
             }
         }
     } finally {
-        this._baseApis._cryptoCallbacks = appCallbacks;
+        // Restore the original callbacks. NB. we must do this by manipulating
```

Comment:
An alternate approach could be for `CrossSigning` to only reference `baseApi` and always pull the callbacks from there. (Either way is fine.)

Reply:
Yeah, I suspect this would be better, really.